### DISCUSSION TITLE
Query MS SQL Server using text field

### DISCUSSION THREAD

**Discussion by sugimoto-atsushi21-fixer** [Budibase/budibase#6652] [Help] [answered]
I want to create a new record using a value input in text fields.

SQL Table (single column)
  (column1 ) 

SQL query
  INSERT INTO table(column1 ) VALUES( {{ value }} )　

{{ value }} is a binding.

I put a Text field in a Form and a Button.
I set id on the Field of the Text field.
I set an Execute Query action which calls the query I defined above on the On Click of the Button and {{ Form.Fields.id }} on the Binding of the Execute query action.

By doing above , I thought that a text input in the Text field is passed to binding of the query but is not.

Please tell me how to pass a text input in the Text field  to the query.

![スクリーンショット 2022-07-12 152456](https://user-images.githubusercontent.com/83997399/178423041-00d13511-e4ce-40d0-b81d-61bb3e132afa.jpg)

**Comment by sugimoto-atsushi21-fixer** [comment, marked as answer]
solved

**Reply by melohagan** [reply]
Hi @sugimoto-atsushi21-fixer Out of curiosity what was your issue and how did you solve it in the end?
